startup.sh

```diff
@@ -15,13 +15,14 @@ fi
 if [ ! -d ~/.mozilla ]
 then
   # sudo dbus-uuidgen > /etc/machine-id \
-  firefox -no-remote -CreateProfile default \
-  && sudo apt update \
-  && sudo apt -y upgrade \
-  && sudo apt -y install /src/GBPCEFwr64.deb
-else
-  sudo /etc/init.d/warsaw start
+  firefox -no-remote -CreateProfile default # \
+  # && sudo apt update \
+  # && sudo apt -y upgrade \
+  # && sudo apt -y install /src/GBPCEFwr64.deb
+#else
+  # sudo /etc/init.d/warsaw start
 fi
+sudo /etc/init.d/warsaw start
 
 /usr/local/bin/warsaw/core \
 && firefox -no-remote -private-window --class CaixaEconomica --name CaixaEconomica https://www.caixa.gov.br
```

Comment:
This file exists and this because is more than process in container. I saw a alternative for init system for container, not remember the name, is very simple than bloated systemd

Reply:
Line 25 not always executed after 'warsaw start' in buster, because the package install script is broken in some way that it returns non-zero status. So it is separated now in install (with workarounds, the '|| true' thing) and start parts.

Reply:
The && must return 0, and its a conditional sentence for shellscripts.
Its also a way to check if worked correctly